src/sdk/namespace_cache.js

```diff
@@ -179,31 +179,32 @@ class NamespaceCache {
         let object_info_cache = null;
         let cache_etag = '';
         const get_from_cache = params.get_from_cache;
-        try {
-            // Remove get_from_cache if exists for maching RPC schema
-            params = _.omit(params, 'get_from_cache');
-            object_info_cache = await this.namespace_nb.read_object_md(params, object_sdk);
-            if (get_from_cache) {
-                dbg.log0('NamespaceCache.read_object_md get_from_cache is enabled', object_info_cache);
-                object_info_cache.should_read_from_cache = true;
-                return object_info_cache;
-            }
+        if (!params.part_number) {
```

Comment:
Here too need to be careful with `part_number===0` which will fall out this condition - but since multipart numbers start from 1 then it should be ok, I would just add a comment to explain it because it might not be easily visible here.